doc/specs/vulkan/appendices/VK_NV_glsl_shader.txt

```diff
@@ -10,7 +10,7 @@ include::meta/VK_NV_glsl_shader.txt[]
 This extension allows GLSL shaders written to the +GL_KHR_vulkan_glsl+
 extension specification to be used instead of SPIR-V.
 The implementation will automatically detect whether the shader is SPIR-V or
-GLSL and compile it appropriatly.
+GLSL, and compile it appropriatly.
```

Comment:
whilst you're here, should be "appropriat**e**ly"

Reply:
yup. Weird, I thought I was fixing that one. Might have been elsewhere. I am gonna do F&R, maybe there are more occurences...